docs/src/masks.md

```diff
@@ -143,30 +143,31 @@ c_no_mask = Fields.Field(FT, ᶜspace_no_mask)
 
 ## Supported operations and caveats
 
-Currently, masks are only supported for `Fields` (and not `DataLayouts`) with
-`SpectralElement2DSpace`s. We do not yet have support for
-`SpectralElement1DSpace`s, and we will likely never offer support for
-`DataLayouts`, as they do not have the space, and can therefore not use the
-mask.
-
-In addition, not all operations support masks. Here is a list of operations of
-supported and unsupported mask operations:
-
- - `DataLayout` operations (`Fields.field_values(f) = 1`) unsupported (these )
- - `fill!` (`@. f = 1`) supported
- - point-wise `copyto!` (`@. f = 1 + z`) supported
- - stencil `copyto!` (`@. ᶜf = 1 + DivergenceF2C()(Geometry.WVector(ᶠf))`) supported (vertical derivatives and interpolations interpolations)
- - spectral element operations `copyto!` (`@. ᶜf = 1 + Operators.Divergence()(ᶠf)`) (unsupported)
+Currently, masked _operations_ are only supported for `Fields` (and not
+`DataLayouts`) with `SpectralElementSpace2D`s. We do not yet have support for
+masked `SpectralElement1DSpace`s, and we will likely never offer masked
+operation support for `DataLayouts`, as they do not have the space, and can
+therefore not use the mask.
+
+In addition, some operations with masked fields skip masked regions
+(i.e., mask-aware), and other operations execute everywhere
+(i.e., mask-unaware), effectively ignoring the mask. Here is a list of
+operations of mask-aware and mask-unaware:
+
+ - `DataLayout` operations (`Fields.field_values(f) = 1`) mask-unaware (will likely never be mask-aware).
+ - `fill!` (`@. f = 1`) mask-aware
+ - point-wise `copyto!` (`@. f = 1 + z`) mask-aware
+ - stencil `copyto!` (`@. ᶜf = 1 + DivergenceF2C()(Geometry.WVector(ᶠf))`) mask-aware (vertical derivatives and interpolations interpolations)
+ - spectral element operations `copyto!` (`@. ᶜf = 1 + Operators.Divergence()(ᶠf)`) mask-unaware
```

Comment:
spectral element operations `copyto!` (`@. f = 1 + Operators.Divergence()(f)`), where `Operators.Divergence` carries out a divergence operation in horizontal directions.  mask-unaware